composer.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "helhum/typoscript-rendering",
+    "type": "typo3-cms-extension",
+    "description": "Can render a TypoScript path by URL, especially useful for Ajax dispatching",
+    "keywords": [
+        "TYPO3",
+        "rendering",
+        "frontend"
+    ],
+    "homepage": "http://insight.helhum.io/post/104880845705/dont-use-eid-ajax-dispatchers-for-your-extbase",
+    "support": {
+        "issues": "https://github.com/helhum/typoscript_rendering/issues"
+    },
+    "authors": [
+        {
+            "name": "Helmut Hummel",
+            "email": "info@helhum.io",
+            "role": "Developer",
+            "homepage": "http://helhum.io"
+        }
+    ],
+    "license": "GPL-2.0+",
+    "require": {
+        "php": ">=5.3.7",
+        "typo3/cms-core": "^6.2 || ^7.6 || ^8.0"
```

Comment:
This extension also works with "inbetween" versions like 7.0, 7.1 etc. I propose to allow this in the dependencies.